src/client/containers/ContextMenuOptions.tsx

```diff
@@ -29,8 +29,15 @@ export const ContextMenuOptions: React.FC<ContextMenuOptionsProps> = ({ clickedN
   const _swapNote = (noteId: string) => dispatch(swapNote(noteId))
   const _swapCategory = (categoryId: string) => dispatch(swapCategory(categoryId))
 
+  const { categories } = useSelector((state: RootState) => state.categoryState)
```

Comment:
Minor refactor - can you update `useSelector((state: RootState) => state.categoryState)` to `useSelector(getSettings)` to match the other selectors? (I just made this refactor yesterday so it probably wasn't like that when you worked on this.)

Reply:
@taniarascia, done—though I assumed you meant to suggest `getCategories` instead. :slightly_smiling_face: